pymeasure/instruments/lakeshore/__init__.py

```diff
@@ -22,6 +22,8 @@
 # THE SOFTWARE.
 #
 
+from .lakeshore_base import LakeShoreTemperatureInputChannel, LakeShoreHeaterOutputChannel
```

Comment:
Maybe leave out that import, because user won't be able to make any use of these classes directly, so it doesn't make much sense for them to be in the main namespace of the manufacturer.

Reply:
Good point, this is removed in the next commit.